### ISSUE TITLE
is it possible to use yours library from Debian by using proot

### ISSUE THREAD

**Issue by JanuszChmiel** [termux/libandroid-shmem#7]
Do you think, that is it possible to use yours library to emulate shared memory while running Debian Buster ARM64 Bit edition by using Termux and Proot? If yes, would you tell Me how could I reach this goal?
If I would like to use yours library from Debian. What would I had to do?
Thank you very much for yours support.

**Comment by ghost** [comment]
This library is not GNU libc compatible. Open issue regarding shared memory support at https://github.com/termux/proot/issues.

Also, I have answered about libandroid-shmem on mailing list:
> It would be complex to maintain version of
https://github.com/termux/libandroid-shmem compatible with GNU libc &
have installable package for each existing Linux distro that can run
under proot. **Maintaining custom package for Linux distributions is out
of scope for Termux devs**.

\- so it is NOT going to be ported to Debian/Ubuntu/etc... at least officially. This is repository for Termux only.

**Comment by JanuszChmiel** [comment]
So if I have understood correctly to yours explanation.
This library is available for Termux and some other developers who are
deeply familiar with C programming would had to customize it for individual
Linux distributions, which can be executed by using Proot from Termux.
I will try to open new issue related to Proot. For beginning to do not
overload MR Benarski too much, it would be A good idea to implement alteast
one call Shmget, because it is used my many applications very frequently.

Including my favourite PCMANFM file manager. Thank you again for yours
continued support. I have made very good decision, that I did not wasted
time while searching custom ROMS to root Android devices. Thanks to Termux
and Proot, I can work securely while not being afraid, that some wrongly
recompiled Kernel will overheat device board ETC. It is The positive result
of yours team work, that you know C and C++ languages so deeply and also
Androidkernel functions. So you are bringing users this perfect environment.
